diode-server/auth/manager_hydra.go

```diff
@@ -226,16 +227,16 @@ func getHydraPagingTokens(response *http.Response, logger *slog.Logger) (string,
 				k, v := strings.TrimSpace(vs[0]), strings.TrimSpace(vs[1])
 				if k == "rel" {
 					switch v {
+					case "first", "\"first\"":
+						first = getHydraPageToken(link, logger)
 					case "next", "\"next\"":
 						next = getHydraPageToken(link, logger)
-					case "prev", "\"prev\"":
-						prev = getHydraPageToken(link, logger)
 					}
 				}
 			}
 		}
 	}
-	return next, prev
+	return first, next
```

Comment:
Should this be `return next, first`?

Reply:
For the context: hydra v25.4.0 replaced offset based pagination with keyset, no more prev page tokens, only first and next